### PR TITLE
Release/2.15.0

### PR DESCRIPTION
**Pull request by jenniferboedker**
2.15.0 (2022-01-24)
----------------------------

**Added**

* Add displayName to ``life.qbic.datamodel.samples.Status`` (#282)

**Fixed**

**Dependencies**

**Deprecated**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
-  <url>http://github.com/qbicsoftware/data-model-lib</url>
-  <description>Data models. A collection of QBiC's central data models and DTOs.</description>
-  <packaging>jar</packaging>
+    <groupId>life.qbic</groupId>
+	<artifactId>data-model-lib</artifactId>
+	<version>2.15.0</version> <!-- <<QUBE_FORCE_BUMP>> -->
+	<name>data-model-lib</name>
+	<url>http://github.com/qbicsoftware/data-model-lib</url>
+	<description>Data models. A collection of QBiC's central data models and DTOs. </description>
+	<packaging>jar</packaging>
```
Comment (KochTobi) [inline pom.xml:14]: The indentation is not correct here. Please revert.

Comment (KochTobi) [inline pom.xml:14]:
```suggestion
  <groupId>life.qbic</groupId>
  <artifactId>data-model-lib</artifactId>
  <version>2.15.0</version> <!-- <<QUBE_FORCE_BUMP>> -->
  <name>data-model-lib</name>
  <url>http://github.com/qbicsoftware/data-model-lib</url>
  <description>Data models. A collection of QBiC's central data models and DTOs. </description>
  <packaging>jar</packaging>
```

**Review by KochTobi — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (jenniferboedker) [reply pom.xml:14]: I did not do it 😅

Comment (jenniferboedker) [reply pom.xml:14]: Its from the other commits in this PR

**Review by KochTobi — APPROVED** [review]
LGTM

**Comment by KochTobi** [comment]
2.15.0 (2022-01-24)
----------------------------

**Added**

* Add displayName to ``life.qbic.datamodel.samples.Status`` (#282)

**Fixed**

**Dependencies**

**Deprecated**

**Pull request merged**